github/actions_variables.go

```diff
@@ -0,0 +1,293 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// Variable represents a repository action variable.
+type Variable struct {
+	Name       string    `json:"name"`
+	Value      string    `json:"value"`
+	CreatedAt  Timestamp `json:"created_at"`
+	UpdatedAt  Timestamp `json:"updated_at"`
+	Visibility string    `json:"visibility,omitempty"`
+	// Used by ListOrgVariables and GetOrgVariables
+	SelectedRepositoriesURL string `json:"selected_repositories_url,omitempty"`
+	// Used by UpdateOrgVariable and CreateOrgVariable
+	SelectedRepositoryIDs SelectedRepoIDs `json:"selected_repository_ids,omitempty"`
+}
+
+// Variables represents one item from the ListVariables response.
```

Comment:
```suggestion
// ActionsVariables represents one item from the ListVariables response.
```

Reply:
I am not sure, given that `type Secrets` has 
```
// Secrets represents one item from the ListSecrets response.
```
comment. Is it ok?

Reply:
Ah, that is unfortunate.  You are saying that "Secrets" should be "ActionsSecrets", correct?
I agree, but let's not make that breaking change here.

Reply:
Both, and everywhere that it is used, please.